app/elements/candidates/invite-candidate-input.html

```diff
@@ -76,7 +67,30 @@
 			_removeItem: function(){
 				this.candidate.Removed = true;
 				this.remove();
-			}
+			},
+            _afterProperties: function(challenges, candidate){
+                var option = document.createElement("option");
+                if (this.challenges.length == 1){
```

Comment:
instead of doing this you can have a 

```html
<template is="dom-if" if="{{isSingleChallenge}}>
     <p>{{challenges[0].Name}}</p>
</template>
<template is="dom-if" if="{{!isSingleChallenge}}>
    <select items="{{challenges}}" on-change="_setChallenge">
        <option value="{{item.Key}}">{{item.Name}}</option>
    </select>
</template>

```